src/installing-workflow/configuring-object-storage.md

```diff
@@ -18,12 +18,26 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
     * Note: If you provide credentials with sufficient access to the underlying storage, Workflow components will create the buckets if they do not exist.
 * **Step 2:** If applicable, generate credentials that have write access to the storage buckets created in Step 1
 * **Step 3:** If you haven't already fetched the helm classic chart, do so with `helmc fetch deis/workflow-beta3`
-* **Step 4:** Open the helm classic chart with `helmc edit workflow-beta3` and look for the template file `tpl/generate_params.toml`
-* **Step 5:** Update the `storage` parameter to reference the storage platform you are using: `s3`, `azure`, `gcs`
-* **Step 6:** Update the values in the section which corresponds to your storage type, including region, bucket names and access credentials
+* **Step 4:** You can update storage details to your deis/workflow-beta3 chart either by using environment variables or by modifying template file `tpl/generate_params.toml`
+    * **1.** Using environment variables:
+        * set `STORAGE_TYPE` to `s3`, `azure`, `gcs` and set the below environment variables accordingly
```

Comment:
this almost seems too long to be in a bulet point form, though I'm not sure how to resolve this.